umm-spec-lib/src/cmr/umm_spec/versioning.clj

```diff
@@ -12,7 +12,7 @@
   {:collection ["1.0" "1.1" "1.2" "1.3" "1.4" "1.5" "1.6" "1.7" "1.8" "1.9" "1.10" "1.11" "1.12" "1.13" "1.14"
                 "1.15" "1.15.1" "1.15.2" "1.15.3" "1.15.4" "1.15.5"
                 "1.16" "1.16.1" "1.16.2" "1.16.3" "1.16.4" "1.16.5" "1.16.6" "1.16.7"
-                "1.17.0" "1.17.1" "1.17.2" "1.17.3"]
+                "1.17.0" "1.17.1" "1.17.2" "1.17.3" "1.18.0"]
```

Comment:
1.18.0 goes a a new line as it is the start of a new series.

Reply:
You mentioned it before I think. I forgot again. Sorry.

Reply:
at least now you know someone is actually reading this.